test/vast_parser.js

```diff
@@ -30,9 +30,10 @@ describe('VASTParser', function() {
       vastParser.on('VAST-resolving', variables =>
         eventsTriggered.push({ name: 'VAST-resolving', data: variables })
       );
-      vastParser.on('VAST-resolved', variables =>
-        eventsTriggered.push({ name: 'VAST-resolved', data: variables })
-      );
+      vastParser.on('VAST-resolved', variables => {
+        delete variables.xml;
```

Comment:
if xml is compared, test failed with memory crash.
In my thought, xml is too deep for compare so i remove it.